core/src/test/java/com/netflix/conductor/core/execution/TestDeciderService.java

```diff
@@ -498,6 +498,7 @@ public void testTaskTimeout() {
         taskType.setName("test");
         taskType.setTimeoutPolicy(TimeoutPolicy.RETRY);
         taskType.setTimeoutSeconds(1);
+        taskType.setTotalTimeoutSeconds(1_000);
```

Comment:
- Why a `1000` second total timeout for this test?
- Is this test covering the changes made to `checkTaskTimeout`?